extensions/README.md

```diff
@@ -25,16 +25,25 @@
 
 #### In progress Khronos and multi-vendor extensions
 
-_Draft Khronos (KHR) extensions are not ratified yet. Multi-vendor extensions do not require ratification,
+_Draft Khronos (KHR) extensions are not ratified yet. Multi-vendor (EXT) extensions do not require ratification,
 but may still change before they are complete._
 
 | Extension | Status |
 |-----------|--------|
-| [KHR_techniques_webgl](2.0/Khronos/KHR_techniques_webgl/README.md) | Feature-complete, seeking feedback. |
-| KHR_compressed_texture_transmission | In development. |
-| [KHR_blend](https://github.com/KhronosGroup/glTF/pull/1302) | In development. |
-| HDR textures ([#1220](https://github.com/KhronosGroup/glTF/issues/1220), [#1365](https://github.com/KhronosGroup/glTF/issues/1365)) | Planned. |
-| Advanced PBR materials ([#1221](https://github.com/KhronosGroup/glTF/issues/1221), [#1079](https://github.com/KhronosGroup/glTF/issues/1079), ...) | Planned. |
+| [KHR_image_ktx2](https://github.com/KhronosGroup/glTF/pull/1612) | Ready for implementation  and feedback. |
```

Comment:
nit:
```suggestion
| [KHR_image_ktx2](https://github.com/KhronosGroup/glTF/pull/1612) | Ready for implementation and feedback. |
```

Reply:
I think we talked about removing this extension completely, so I'm not sure I would call this ready for implementation.

Reply:
Oh! I seem to have missed that, and it's still marked as a dependency of other extensions in https://github.com/KhronosGroup/glTF/pull/1612. What's the alternative?

Reply:
That PR hasn't been changed since we talked about the updates.

Reply:
Understood, but I missed that and don't know what's changed. Are you suggesting I just remove this line? Or should anything else related to #1612 be changed here? Will each extension independently permit `.ktx2` images, without `KHR_image_ktx2`?

Reply:
I would just change the status to In Development instead of Ready for implementation...

Reply:
Done. 👍